js-basics_ex-2/dataParser2.mjs

```diff
@@ -0,0 +1,150 @@
+class SalesDataParser {
+  header = [];
+  values = [];
+
+  constructor(data) {
+    this.header = data[0];
+    this.values = data.slice(1);
+  }
+
+  salesGroup(sales_column, groupArray) {
```

Comment:
```suggestion
 addGroupColumn(sourceColumn, groupArray) {
```

Reply:
I have implemented the changes based on the feedback you provided in my code. Please review and let me know if any further adjustments are needed. Thank you!